README.md

```diff
@@ -93,6 +93,19 @@ For projects hosted by [GitLab](https://gitlab.com) reference [Microsoft.SourceL
 </ItemGroup>
 ```
 
+If your project is hosted by GitLab older than version 12.0 you must specify `SourceLinkGitLabHost` item group in addition to the package reference:
+
+```xml
+<ItemGroup>
+  <SourceLinkGitLabHost Include="gitlab.yourdomain.com" Version="11.0"/>
```

Comment:
Is the property used to set the hostname really called Include?

Reply:
Yes. `Include` adds item to `ItemGroup`.